frontend/src/main-components/CreateCardContent.js

```diff
@@ -1,22 +1,134 @@
-import React from "react";
+import React, { Component } from "react";
 import styled from "@emotion/styled";
+import css from "./CreateCardContent.css";
+import { motion } from "framer-motion";
+import FrontCard from "../flashcards/FlashcardFrontPreview";
+import BackCard from "../flashcards/FlashcardBackPreview";
+import { getTranslation } from "../sub-components/translate";
+import LangaugeScroll from "../sub-components/languageScroll";
+class CreateCardContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      imgSrc: "",
+      text: "none",
+      translation: "none",
+      fromLang: "none",
+      toLang: "none",
+    };
+    this.TranslateText = this.TranslateText.bind(this);
+    this.fromLangSelected = this.fromLangSelected.bind(this);
+    this.toLangSelected = this.toLangSelected.bind(this);
+  }
+  /** 
+   * When the user has finished typing the Google translate
+   * API iss called to fetch the translated version    of text
+   */
+  TranslateText(event) {
+    event.preventDefault();
+    console.log(this.state.fromLang);
```

Comment:
Ideally we would remove this before we deploy our app, but for now we can keep in the print statements for testing purposes.